src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+form {
+  position: relative;
+  margin: 20px 7px;
+}
+
+.input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
```

Comment:
You declared in `@font-face` font-family called 'avenir' (starting with small letter), but here you try to use font-family 'Avenir' (starting with a capital letter). Css will not know such font-family (if it's not installed on your computer), it knows only 'avenir' which you have declared in font-face. So better rename it in font-face to avoid unexpected fonts